### ISSUE TITLE
UncertaintyInterface improvements

### ISSUE THREAD

**Issue by jamesdolezal** [jamesdolezal/slideflow#148]
## Feature
Currently, UncertaintyInterface appends uncertainty estimates to the list of logits, which is problematic for multiple reasons and not intuitive for users. Uncertainty estimates should be returned as separate tuples. Similarly, Heatmaps should store uncertainty estimates separately from logits.

**Comment by jamesdolezal** [comment]
Fixed in 73de4c8
